src/Notes/AddFirstProduct.php

```diff
@@ -47,6 +47,16 @@ public static function get_note() {
 			return;
 		}
 
+		// Don't show if there is an orders.
+		$args   = array(
```

Comment:
I like that you kept the query here pretty "open" so it will capture the edge case of there being prior orders that may have been refunded, or canceled. Any order object presence would imply the site has setup a product at some point 👍